### PR TITLE
Improved: Show Completed Shipments Along with Open Shipments (#380)

### PR DESCRIPTION
**Pull request by amansinghbais**
### Related Issues

#380

### Short Description and Why It's Useful

- Showed completed shipments along with the open shipments.
- Updated shipment details page for the completed shipments.

### Screenshots of Visual Changes before/after (If There Are Any)

Before
Screenshot from 2024-08-07 18-07-34

After
Screenshot from 2024-08-07 18-07-13
Screenshot from 2024-08-07 18-07-08

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/hotwax/receiving#contribution-guideline)

### CODE REVIEW

File: src/views/Shipments.vue [hunk 1]
```diff
@@ -154,4 +175,12 @@ export default defineComponent({
...
   }
 })
-</script>
\ No newline at end of file
+</script>
+
+<style scoped>
+@media (min-width: 991px) {
+  ion-header > div {
+    display: flex;
```
Comment (dt2patel) [inline src/views/Shipments.vue:183]: @amansinghbais why not use the same pattern as job manager?

File: src/views/Shipments.vue [hunk 2]
```diff
@@ -104,7 +122,7 @@ export default defineComponent({
       const payload = {
         "inputFields": {
           "destinationFacilityId": this.user.facilityId,
-          "statusId": "PURCH_SHIP_SHIPPED",
+          "statusId": this.selectedSegment === "open" ? "PURCH_SHIP_SHIPPED" : "PURCH_SHIP_RECEIVED",
```
Comment (dt2patel) [inline src/views/Shipments.vue:125]: Should this be available on all pages instead of just the shipments page? I think so right? @amansinghbais

**Review by dt2patel — CHANGES_REQUESTED** [review]
Please take a look at my comments

File: src/views/Shipments.vue (hunk 1 above)
Comment (amansinghbais) [reply src/views/Shipments.vue:183]: @dt2patel sir, The same pattern was used in the pipeline page of Job Manager app. Took reference from there only.

File: src/views/Shipments.vue (hunk 2 above)
Comment (amansinghbais) [reply src/views/Shipments.vue:125]: Yes, we can extend this approach to other pages as well to ensure consistency across the application. Initially, the implementation was focused on the shipment page as discussed, but we can apply the same for other pages as well.

File: src/store/modules/order/actions.ts [hunk 3]
```diff
@@ -71,20 +71,6 @@ const actions: ActionTree<OrderState, RootState> = {
   async getOrderDetail({ commit, state }, { orderId }) {
     let resp;
 
-    const current = state.current as any
```
Comment (ravilodhi) [inline src/store/modules/order/actions.ts:74]: This code is removed to fetch the data every time when we go to detail page, right?

File: src/views/ReturnDetails.vue [hunk 4]
```diff
@@ -157,10 +157,8 @@ export default defineComponent({
   },
   async ionViewWillEnter() {
     const current = await this.store.dispatch('return/setCurrent', { shipmentId: this.$route.params.id })
-
-    if(!this.isReturnReceivable(current.statusId)) {
-      showToast(translate("This return has been and cannot be edited.", { status: current?.statusDesc?.toLowerCase() }));
-    }
+    console.log(this.current);
```
Comment (ravilodhi) [inline src/views/ReturnDetails.vue:160]: Is this console log intentional?

Comment (amansinghbais) [reply src/views/ReturnDetails.vue]: No this is left by mistake, Reverted.

File: src/store/modules/order/actions.ts (hunk 3 above)
Comment (amansinghbais) [reply src/store/modules/order/actions.ts:74]: Yes, so as to avoid old item status issue due to slow Solr updation.

**Pull request merged**